src/main/java/com/gameList/gamelist/model/Xbox.java

```diff
@@ -0,0 +1,53 @@
+package com.gameList.gamelist.model;
+
+public class Xbox {
+	private int id;
+	private String gameName;
+	private String price;
+	private String console;
+	
+	public Xbox (int id, String gameName, String price, String console) {
+		this.setId(id);
+		this.setGameName(gameName);
+		this.setPrice(price);
+		this.setConsole(console);
+	}
+	
+	public int getId() {
+		return id;
+	}
+	
+	public void setId(int id) {
+		this.id = id;
+	}
+	
```

Comment:
I created a new DTO class - Xbox - to be used in the controller to add mock data and display on the HTML page